### PR TITLE
AEA-1730 Migrate EPSAT release interaction from flask to hapi

### PR DESCRIPTION
**Pull request by booshibooshi**

### CODE REVIEW

File: tool/site/server_hapi/src/services/communication/mock-eps-client.ts [hunk 1]
```diff
@@ -50,6 +50,30 @@ export class MockEpsClient implements EpsClient {
...
+    const url = `https://${process.env.APIGEE_DOMAIN_NAME}/electronic-prescriptions/FHIR/R4/Task/$release`
+    const statusCode = 200
+    //TODO - why is the mock client sending real requests?
+    const spineResponse = (await axios.post(url, body, {
+      headers: {
+        "X-Request-ID": uuid.v4(),
+        "X-Raw-Response": "true"
+      }
+    })).data
+    const spineResponseStr = typeof spineResponse === "string" ? spineResponse : JSON.stringify(spineResponse)
```
Comment (mfjarvis) [inline tool/site/server_hapi/src/services/communication/mock-eps-client.ts:63]: The live client doesn't stringify the response if it's an object, so I think we can just return spineResponse here too

File: tool/site/server_hapi/src/services/communication/live-eps-client.ts [hunk 2]
```diff
@@ -32,6 +32,18 @@ export class LiveEpsClient implements EpsClient {
     return {statusCode, fhirResponse, spineResponse}
   }
 
+  async makeReleaseRequest(body: Parameters): Promise<EpsResponse<Bundle | OperationOutcome>> {
+    const requestId = uuid.v4()
+    const rawResponseHeaders = {
+      "x-raw-response": "true"
+    }
+    const response = await this.makeApiCall<OperationOutcome>("Task/$release", body, requestId)
```
Comment (mfjarvis) [inline tool/site/server_hapi/src/services/communication/live-eps-client.ts:40]: `Bundle | OperationOutcome`? Not that it matters evidently!

**Review by mfjarvis — APPROVED** [review]

File: tool/site/server_hapi/src/services/communication/live-eps-client.ts (hunk 2 above)
Comment (booshibooshi) [reply tool/site/server_hapi/src/services/communication/live-eps-client.ts:40]: good spot

Comment (booshibooshi) [reply tool/site/server_hapi/src/services/communication/live-eps-client.ts:40]: updated on next pr

File: tool/site/server_hapi/src/services/communication/mock-eps-client.ts (hunk 1 above)
Comment (booshibooshi) [reply tool/site/server_hapi/src/services/communication/mock-eps-client.ts:63]: updated on next PR

**Pull request merged**